internal/faustjs.org/docs/blocks/gutenberg-blocks.mdx

```diff
@@ -0,0 +1,55 @@
+---
+slug: gutenberg-blocks-provider
+title: Getting Started with Gutenberg Blocks Provider
+---
+
+# Getting Started with Gutenberg Blocks Provider
+
+## Quick Start
+
+:::note
+
+Make sure you have completed the initial setup for Faust at [Getting Started](/docs/getting-started).
+
+:::
+
+Install the blocks:
+
+```bash
+npm i @faustwp/blocks
+```
+
+Open ```_app.js``` and import the blocks provider:
+
+```js
+import { WordPressBlocksProvider } from '@faustwp/blocks';
+```
+
```

Comment:
Also please provide a really simple block implementation and explain what it does. For example:

```
import { gql } from '@apollo/client';
import React from 'react';

import getStyles from '../utilities/getStyles.js';

export default function CoreParagraph(props) {
  const attributes = props.attributes;
  return (
    <p
      className={attributes?.cssClassName}
      dangerouslySetInnerHTML={{ __html: attributes.content }}></p>
  );
}

CoreParagraph.fragments = {
  entry: gql`
    fragment CoreParagraphFragment on CoreParagraph {
      attributes {
        content
        fontSize
        textColor
        backgroundColor
        cssClassName
      }
    }
  `,
  key: `CoreParagraphFragment`,
};
```

Reply:
index.js

```
import CoreParagraph from './CoreParagraph';
export default {
  CoreParagraph,
};
```